components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1810,4 +1809,27 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
     }
     //
     //#endregion
+
+
+
+    //#region Projects concerns
+    //
+    async getProviderRepositoriesForUser(params: { provider: string }): Promise<ProviderRepository[]> {
```

Comment:
this is implemented in EE space because the github-app.ts is also there. 

@svenefftinge, it's ok to move the github-app.ts to the non-EE sources?

Reply:
Yes, that is fine